scripts/setup_uberspace.sh

```diff
@@ -30,6 +30,14 @@ fi
 # Pull env into current shell
 source .env
 
+echo 'Setting up initial data for MariaDB'
+mariadb < docker-entrypoint-initdb.d/001-init.sql
+
+if [ "$ENVIRONMENT" = 'production' ]; then
+  echo 'Making data empty in the MariaDB table lti_entity'
+  mariadb serlo -e 'TRUCATE table lti_entity'
+fi
```

Comment:
This seems a little bit dangerous to me. Running `setup_uberspace.sh` on Uberspace 'production' would delete all content. We have backups but still. Do you have an idea how we could prevent this accident from happening?

Reply:
You are right.
Let's just delete this lines. Production will have then the dummy data, but it is just one entry and harmless.

Reply:
I think production having one dummy element is fine. That is a nice and easy solution. :slightly_smiling_face: